Lib/elements/base_element.py

```diff
@@ -0,0 +1,33 @@
+"""Class for handling base element on the page."""
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+class BaseElement(object):
+    def __init__(self, driver, element_tuple, timeout=10):
+        self.driver = driver
+        self.locator_type = element_tuple[0]
+        self.locator = element_tuple[1]
+        self.timeout = timeout
+
+    def click(self):
+        """Click action."""
+        self.element = self.find_visible_element(self.locator_type, self.locator, self.timeout)
+        self.element.click()
+
+    def get_text(self):
+        """Get element text.
+
+        Returns:
+            str: text of the element.
+        """
+        element = self.find_visible_element(self.locator_type, self.locator, self.timeout)
+        return element.text
+
+    def find_visible_element(self, locator_type, locator, timeout=None):
+        wait = WebDriverWait(self.driver, 10)
```

Comment:
it is probably better to use variable like timeout instead of hard coding value (10)